conversation_classification/kaggle/model.py

```diff
@@ -127,15 +180,34 @@ def estimator_spec_for_softmax_classification(logits, labels, mode):
       loss=loss,
       train_op=train_op,
       training_hooks=[logging_hook],
-      predictions={'loss': loss}
+      predictions={'loss': loss},
+      export_outputs=export_outputs
     )
 
   # EVAL Mode
   eval_metric_ops = {
-    'accuracy': tf.metrics.accuracy(labels=labels, predictions=predicted_classes)
+    'accuracy': tf.metrics.accuracy(
+      labels=labels, predictions=predicted_classes),
+    'auc': tf.metrics.auc(labels=labels, predictions=predicted_classes),
+    'mean_per_class_accuracy': tf.metrics.mean_per_class_accuracy(
```

Comment:
Isn't this always the same as accuracy as you're treating this as a binary classification problem?

Reply:
Ha, yes. I was just adding all the metrics I could. I'll take it out as it is totally redundant.